examples/efs/filesystem_encrypted.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: efs.aws.upbound.io/v1beta1
+kind: FileSystem
+metadata:
+  name: example
+  labels:
+    testing.upbound.io/example-name: efs
+spec:
+  forProvider:
+    region: us-west-1
```

```suggestion
    region: us-west-1
    encrypted: true
```
When specifying `kms_key_id`, encrypted needs to be set to true: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/efs_file_system#kms_key_id